src/components/TextWithCopy/index.tsx

```diff
@@ -12,7 +12,6 @@ import { Txt, TxtProps } from '../Txt';
 const Wrapper = styled(Txt.span)<InternalTextWithCopyProps>`
```

Comment:
I suppose that once we add the property to the Txt component, this file can be deleted